### ISSUE TITLE
Very slow and selenium is not working

### ISSUE THREAD

**Issue by haxsysgit** [Jookie262/animepahe-auto-downloader#3]
i tried to run the project but i think the chrome driver is outdated and can you make it to support firefox
check https://github.com/haxsysgit/autopahe

**Comment by Jookie262** [comment]
Yeah, you're right. I didn't think about anything else except for making my life easier to download anime from Animepahe. It would be awesome if we could update the chrome driver and also make it work with Firefox. The link you sent is really impressive. I had no idea your project was so much better than mine. Thanks for sharing it with me.
